docs/components/concepts/clusters.md

```diff
@@ -6,72 +6,88 @@ description: "Learn more about the clusters available in your Camunda 8 plan."
-- **Starter** plan customers are limited based on the [fair usage limits of the plan](https://camunda.com/legal/fair-usage-limits-for-starter-plan/).
+When [creating a cluster](/components/console/manage-clusters/create-cluster.md), you can customize the cluster **type** and **size** to meet your organization's availability and scalability needs, and to provide control over cluster performance, uptime, and disaster recovery guarantees.
 
-Production clusters come in three sizes: small (S), medium (M), and large (L). To learn more about the size of cluster best suited for your use case, refer to our [Best Practices](/components/best-practices/best-practices-overview.md) for more information on [sizing your runtime environment](/components/best-practices/architecture/sizing-your-environment.md#sizing-your-runtime-environment).
+:::note
 
-The following table shows each plan and available type or size of cluster:
+Prior to 8.6, clusters were configured by hardware size (S, M, L).
 
-|            | Development | Production - S | Production - M | Production - L |
-| ---------- | ----------- | -------------- | -------------- | -------------- |
-| Free Trial | \-          | X              | \-             | \-             |
-| Free       | \-          | \-             | \-             | \-             |
-| Starter    | X           | X              | \-             | \-             |
-| Enterprise | X           | X              | X              | X              |
+- To learn more about clusters prior to 8.6, see previous documentation versions.
+- To learn more about migrating your existing clusters to the newer model, contact your Customer Success Manager.
 
-When you deploy and execute your [BPMN](/components/modeler/bpmn/bpmn.md) or [DMN](/components/modeler/dmn/dmn.md) models on a production cluster, this might impact your monthly (Starter) or annual (Enterprise) total fee, meaning the more you execute your models, the higher your total fee may be.
+:::
 
-## Free Trial cluster
+## Cluster type
 
-Free Trial clusters have the same functionality as a production cluster, but are size [small (S)](/components/best-practices/architecture/sizing-your-environment.md#camunda-8-saas) and only available during your trial period. You cannot convert a Free Trial cluster to a different kind of cluster.
+The cluster type defines the level of availability and uptime for the cluster.
 
-Once you sign up for a Free Trial, you are able to create one production cluster for the duration of your trial.
+For example:
 
-When your Free Trial plan expires, you are automatically transferred to the Free Plan. This plan allows you to model BPMN and DMN collaboratively, but does not support execution of your models. Any cluster created during your trial is deleted, and you cannot create new clusters.
+- Use a **Basic** cluster for experimentation, early development, and basic use cases not requiring a guaranteed high up-time.
```

Comment:
Felt like: If we give two examples, why not give all three :D

Reply:
Agreed - something like the following?

* Use a Standard cluster for production-ready uses cases not requiring a guaranteed high uptime.

Reply:
Added in as a third bullet